src/importexport/mei/internal/meiimporter.cpp

```diff
@@ -1254,6 +1254,16 @@ bool MeiImporter::readSectionElements(pugi::xml_node parentNode)
 {
     bool success = true;
 
+    libmei::Section meiSection;
+    meiSection.Read(parentNode);
+
+    if (meiSection.HasRestart() && meiSection.GetRestart() == libmei::BOOLEAN_true) {
+        Measure* measure = static_cast<Measure*>(m_score->measures()->last());
```

Comment:
It would be better to check whether this list is empty or not (to avoid crashing on an empty score). Also, this list stores MeasureBase*, which means it can contain only 1 text frame element, so this static_cast should be replaced with dynamic_cast (or you need to explicitly check the type)...